gatsby-config.js

```diff
@@ -159,5 +159,18 @@ module.exports = {
     },
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-catch-links',
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: 'React Docs',
+        short_name: 'React',
+        start_url: '/',
+        background_color: '#20232a',
+        theme_color: '#20232a',
+        display: 'standalone',
+        icon: 'static/logo-512x512.png',
```

Comment:
What's this `icon` actually being used for at this point? Can we remove it, given that you've added the static 180x180 icon and that's the only one referenced in the `<link>`?

Reply:
the one added in `<link>` is for apple. Apparently, some versions of apple do not read icons from webapp manifest so they need a special tag in html head but Chrome and other browsers use webapp manifest for icons.

So we'll still need `icon` for chrome

Reply:
> So we'll still need `icon` for chrome

Ah, is this the icon shown as the splash screen when the app is opening?

TBH I'm not sure if this change is a net positive for Chrome. It's nice that the default title is just "React", and the splash screen is nice- but the "Add to homescreen" action takes a couple of seconds (during which I see a loading spinner) now instead of just being instant (like it is on reactjs.org, without this change) -

![Screenshot_20190930-140116_Chrome](https://user-images.githubusercontent.com/29597/65917639-b26e2400-e38c-11e9-9602-04aa6eaab034.jpg)

Reply:
`icon` takes that image and generates multiple images so `512x512` is used in homescreen and `192x192` is usually used to create that logo on android and other sizes are for different screen sizes I guess.

I am not sure but I think the reason logo loads fast without `icon` is that it uses `favicon.ico` which is already loaded when you open website but when you specifically define icon it probably loads when you click 'Add to Homescreen'.

Also, we can't really rely on favicon to provide logo since it does not work in some mobiles (does not work in mine at least)
<img src="https://user-images.githubusercontent.com/30949385/65901564-19fc9180-e3d6-11e9-97ae-a3d1c7aa864a.png" width="200">

Reply:
I don't really know much about this particular stuff. I'd love for someone else to weigh in.

Reply:
Cool, thank you for the quick response :)